src/spanningTree/prim.cpp

```
@@ -0,0 +1,65 @@
+
+ ********************************************************************PGR-GNU*/
+
+#include "spanningTree/prim.hpp"
+
+#include <vector>
+#include <set>
+
+namespace pgrouting {
+namespace functions {
+
+std::vector<MST_rt>
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)